src/components/Input/Input.test.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import '@testing-library/jest-dom';
+
+import Input from 'components/Input';
+
+describe('Input component', () => {
+  test('should render correctly without props', () => {
+    render(<Input />);
+
+    expect(screen.getByTestId('input')).toBeInTheDocument();
+  });
+  test('should have correct id and label when placeholder is passed', () => {
+    const testPlaceholder = 'test-input';
+    render(<Input placeholder={testPlaceholder} />);
+
+    expect(screen.getByTestId('input')).toBeInTheDocument();
+    expect(screen.getByTestId('input')).toHaveAttribute('id', testPlaceholder);
+    expect(screen.getByTestId('input-label')).toBeInTheDocument();
+    expect(screen.getByTestId('input-label')).toHaveTextContent(
```

Comment:
We could assign `screen.getByTestId('input')` and `screen.getByTestId('input-label')` to variables to avoid duplication search.

Reply:
good point, done